test/e2e/framework/podlogs/podlogs.go

```diff
@@ -251,7 +251,7 @@ func WatchPods(ctx context.Context, cs clientset.Interface, ns string, to io.Wri
 						)
 					} else if cst.State.Running != nil {
 						fmt.Fprintf(buffer, "RUNNING")
-					} else if cst.State.Waiting != nil {
+					} else if cst.State.Terminated != nil {
```

Comment:
could you please explain this change?

Reply:
The following print string is `fmt.Fprintf(buffer, "TERMINATED: %s - %s",`. So it should check the `Terminated` state instead of the `Waiting` state which already be checked above.

Reply:
Yeah, "Waiting" is also already handled above